.github/workflows/docker_snapshot.yml

```diff
@@ -50,8 +50,9 @@ jobs:
       - name: Build hazelcast with Maven
         run: |
           mvn -f hazelcast/pom.xml -B clean install -DskipTests
-          rm hazelcast/hazelcast-all/target/*-sources.jar
-          cp hazelcast/hazelcast-all/target/hazelcast-all-*.jar hazelcast-all.jar
+          rm hazelcast/hazelcast/target/*-sources.jar \
+            hazelcast/hazelcast/target/*-tests.jar || true
+          cp hazelcast/hazelcast/target/hazelcast-*.jar hazelcast.jar
```

Comment:
Just to understand better, with this change, for 4.2.z, we are going to publish `hazelcast` JAR, right? But previously, it was `hazelcast-all` JAR.

Not sure how those snapshot images are used but isn't this a behavior change for 4.y.z?

Reply:
These snapshot images are our internal images used e.g. for regular dynamic security scans.
There should be no problem related to the `hazelcast-all -> hazelcast` change.

Reply:
Oh, OK then. Thanks for the explanation